conda_forge_tick/update_recipe/version.py

```diff
@@ -557,15 +596,27 @@ def update_version_feedstock_dir(
         )
 
 
-def _update_version_feedstock_dir_local(feedstock_dir, version, hash_type):
-    with open(os.path.join(feedstock_dir, "recipe", "meta.yaml")) as f:
-        raw_meta_yaml = f.read()
-    updated_meta_yaml, errors = update_version(
-        raw_meta_yaml, version, hash_type=hash_type
-    )
+def _update_version_feedstock_dir_local(
+    feedstock_dir, version, hash_type
+) -> (bool, set):
+    feedstock_path = Path(feedstock_dir)
+
+    recipe = feedstock_path / "recipe" / "meta.yaml"
+    if recipe.exists():
```

Comment:
use the same pattern here
```suggestion
    recipe_path = None
    recipe_path_v0 = feedstock_path / "recipe" / "meta.yaml"
    if recipe_path_v0.exists():
        recipe_path = recipe_path_v0
```